entity-framework/core/miscellaneous/connection-strings.md

```diff
@@ -93,3 +93,4 @@ public void ConfigureServices(IServiceCollection services)
         options.UseSqlServer(Configuration.GetConnectionString("BloggingDatabase")));
 }
 ````
+Note: the following using statement is required: using Microsoft.EntityFrameworkCore;
```

Comment:
@Rick-Anderson I don't like the using-in-comment approach, as it's not really a code comment that explains the code it's next to and is included in that place only to provide information to the reader of an article.  If you need to mention the using I'd put it in the text.  If you want to show it in a code snippet I think it's better to show it as a using statement in a separate immediately preceding snippet.